pkgs/development/python-modules/dateparser/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage, isPy3k
+, nose
+, nose-parameterized
+, mock
+, glibcLocales
+, six
+, jdatetime
+, pyyaml
+, dateutil
+, umalqurra
+, pytz
+, tzlocal
+, regex
+, ruamel_yaml }:
+buildPythonPackage rec {
+  name = "dateparser-${version}";
```

Comment:
Just one more tiny change.
```
pname = "dateparser";
version = "0.6.0";
```
and drop `name`.

Reply:
ah, that works now! I remember that name was always necessary.